src/gocept/amqprun/tests/test_server.py

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) 2010-2012, 2020 gocept gmbh & co. kg
 # See also LICENSE.txt
```

Comment:
You have removed the license hint in other files, maybe you could do that here too.

Reply:
Okay, removing all © headers.